web_readonly_bypass/README.rst

```diff
@@ -0,0 +1,86 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Can you take a look to the latest version of the sample README.rst (https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst) and fix the following issues:
* add target to the image licence badge
* put a lot of '===' above the module title to make the title a real title
* add a link 'Try me on Runbot' in the usage section